scipy/stats/_continuous_distns.py

```diff
@@ -8414,6 +8414,12 @@ def _ppf(self, q, c):
     def _entropy(self, c):
         return np.log(2*np.pi*(1-c*c))
 
+    def _fitstart(self, data):
+        # Use 0.5 as the initial guess of the shape parameter.
+        # For the location and scale, use the minimum and
+        # peak-to-peak/(2*pi), respectively.
+        return 0.5, np.min(data), np.ptp(data)/(2*np.pi)
```

Comment:
Makes sense, as 0.5 is the middle of the allowed interval rather than right end, min(data) ensures that all data is within the support, and p2p looks reasonable for scale, although I don't know what's appropriate for these sorts of distributions.  But better than the default, I'll bet.